agent/src/main/java/com/teamscale/jacoco/agent/util/DebugLogDirectoryPropertyDefiner.java

```diff
@@ -0,0 +1,17 @@
+package com.teamscale.jacoco.agent.util;
+
+import java.nio.file.Path;
+
+/** Defines a property that contains the path to which log files should be written. */
+public class DebugLogDirectoryPropertyDefiner extends LogDirectoryPropertyDefiner {
+
+	/* package */ static Path filePath = null;
+
+	@Override
+	public String getPropertyValue() {
+		if (filePath == null) {
+			return super.getPropertyValue();
+		}
+		return filePath.resolve("logs").toAbsolutePath().toString();
+	}
```

Comment:
Not sure if this is the right place to do it, but we should handle the case when its either a wrong path or the agent is not allowed to log to the path

Reply:
I added a check whether the file path is valid, but how can I check whether the file path can be written to?

Reply:
Not sure about that rn. Maybe thats to much of an overhead. Can you check whats happening when you define a not existing or not writeable path? If thats handled gracefully we can skip the validation here

Reply:
In case of an invalid file path, we print the exception to the console and then continue running normally. But logging is then only done to the console.
Another thing we can do is pass null as the logging path. The logger will then create a new directory named `debugLogDirectory_IS_UNDEFINED`. Which one do you prefer?

Reply:
The first option is ok